Natalia_Niemczyk_LaTeX.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[]{article}
+\usepackage{qtree}
+\usepackage{phonrule}
+\usepackage{tipa}
+\usepackage{ulem}
+\usepackage{starfont}
+\usepackage{hieroglf}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\title{\LaTeX Homework}
+\author{Natalia Niemczyk}
+\date{May 2019}
+\begin{document}
+\maketitle
+\tableofcontents
+\section{My favorite linguistics and computer science topics}
+\begin{enumerate}
+    \item Sentiment analysis
+    \item Web design    
+    \item Game development
+    \item Sociolinguistics
+    \item Phonology    
+    \item Cognitive linguistics
+\end{enumerate}
+\section{About me}
+\textipa{/maI neIm Iz \|[na\|[talja \ae nd aI stAdi n\ae tSr@l l\ae NgwIdZ pr@UsEsIN In g\|[daN\|[sk ||
+\newline aI h\ae v O:lweIz IndZOId lE:nIN nju: l\ae NgwIdZIz | IspES@li INglIS ||\newline
+aI TINk pr@Ugr\ae mIN Iz IntrIstIN \ae nd k\ae n bi: @ greIt aUtlEt fO: kri:eItIvIti ||\newline
+in\|[tErE\|[sujE \|[siE ruv\|[njE\:z \:s\|[tuk\~O | f \:s\texttoptiebar{t\:s}Egul\|[nOC\texttoptiebar{tC}ji fO\|[tOgrafj\~O Ora\|[s ilu\|[s\|[tra\texttoptiebar{ts}j\~O ||\newline

```

Comment:
1. się > pierwsza spółgłoska miękka :)
2. również > palatal nasal
3. \:s for Polish <sz> does not work... please use \textrtails for all cases